ptero_petri/api/wsgi.py

```diff
@@ -9,11 +10,13 @@
 
 def parse_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--port', type=int, default=5000)
     parser.add_argument('--debug', action='store_true', default=False)
     return parser.parse_args()
 
 
 if __name__ == '__main__':
     args = parse_args()
-    app.run(port=args.port, debug=args.debug)
+    app.run(
+        host=os.environ.get('PTERO_PETRI_HOST'),
+        port=os.environ.get('PTERO_PETRI_PORT'),
```

Comment:
In .tox/py27/lib/python2.7/site-packages/flask/app.py
722     def run(self, host=None, port=None, debug=None, **options):

So we will want to use square bracket notation here.